### PR TITLE
V2024.02.0 RC

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR is the release candidate for the v2024.02.0 release.

Note: the companion PR that only shows changes between the RC branch and develop is https://github.com/LLNL/RAJA/pull/1599. Please review that PR and **approve this one to merge.**

### CODE REVIEW

**Review by MrBurmark — APPROVED** [review]

**Comment by rhornung67** [comment]
@artv3 @rchen20 @davidbeckingsale @mdavis36 I'm going to merge this and put out the RAJA release later today. If you would like to review this first, please look it over soon.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,67 +1,84 @@
 ###############################################################################
-# Copyright (c) 2016-22, Lawrence Livermore National Security, LLC
+# Copyright (c) 2016-23, Lawrence Livermore National Security, LLC
```
Comment (rchen20) [inline .gitlab-ci.yml:2]:
```suggestion
# Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
```

**Review by rchen20 — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (rhornung67) [reply .gitlab-ci.yml]: Ooops! Thank you. Fixed

**Review by artv3 — APPROVED** [review]

**Review by adayton1 — APPROVED** [review]
LGTM

**Pull request merged**